CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -23,6 +23,12 @@ Reading the text below is not necessary to create a new quest. Duplicating an ex
 
 If you are doing it for the first time, don't worry if there is an error to solve along the way, this is typical for setting up an Android development environment. See [CONTRIBUTING file](CONTRIBUTING.md#development) which has some links to information about the setup.
 
+## Alternative to Android Studio
+
+- Instead of installing Android Studio locally, one can use GitHub Actions to build `.apk` files. Click on `Actions` > `Build debug apk` on your GitHub fork, and after about 15 minutes, download the ready-made `debug-apk.zip` which contains `.apk` file to install on your phone manually.
```

Comment:
```suggestion
- Instead of installing Android Studio locally, one can use GitHub Actions to build `.apk` files. Click on `Actions` > `Build debug apk` on your GitHub fork, and after about 15 minutes, download the ready-made `debug-apk.zip` which contains an `.apk` file to install on your phone manually.
```